WeightProducer/python/MCSampleValues.py

```diff
@@ -1805,7 +1660,9 @@ def get_value(self, name, energy, year, key, strict=False):
         """
         fields = [self.__key_field_map[key][0]+"_"+energy,self.__key_field_map[key][0]+"_"+year]
         if not name in self.__values_dict:
-            raise KeyError("ERROR MCSampleValuesHelper::Unknown process \"" + str(name) + "\"")
+            print("ERROR MCSampleValuesHelper::Unknown process \"" + str(name) + "\"")
```

Comment:
why does this no longer raise an exception?

Reply:
Ah, that is an oversight on my part when I pushed my changes onto the branch. I will restore the exception raising.

Reply:
Reverted in 4d2e518